READMEs/simple-remote.md

```diff
@@ -0,0 +1,100 @@
+<!--
+Copyright 2022 Ocean Protocol Foundation
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Quickstart: _Remotely_ Publish Data NFT & Datatoken
+
+This quickstart is like the [simple-flow](READMEs/data-nfts-and-datatokens-flow.md), but uses [remote services and Mumbai testnet](https://docs.oceanprotocol.com/core-concepts/networks#mumbai).
+
+Let's go through each step.
+
+## 1. Setup
+
+### Prerequisites & installation
+
+From [simple-flow](data-nfts-and-datatokens-flow.md), do:
+- [x] Setup : Prerequisites
+- [x] Setup : Install the library
+
+
+### Create Mumbai Accounts (One-Time)
+
+From [get-test-MATIC](get-test-MATIC.md), do:
+- [x] Create two new accounts
+- [x] Get (fake) MATIC
+
+### Create Config File for Services
+
+In your working directory, create a file `config.ini` and fill it with the following. It will use pre-existing services running for mumbai testnet.
+
+```text
+[eth-network]
+network_name = mumbai
+network = https://rpc-mumbai.maticvigil.com
+address.file = ~/.ocean/ocean-contracts/artifacts/address.json
+block_confirmations = 0
+
+[resources]
+metadata_cache_uri = https://v4.aquarius.oceanprotocol.com
+provider.url = https://v4.provider.mumbai.oceanprotocol.com
+```
+
+### Set envvars
+
+In the console:
+```console
+# For services: point to config file
+export OCEAN_CONFIG_FILE=config.ini
+
+# For services: ensure no other envvars that override config file values
+unset OCEAN_NETWORK_URL METADATA_CACHE_URI AQUARIUS_URL PROVIDER_URL
+
+# For accounts: set private keys
+export TEST_PRIVATE_KEY1=<your TEST_PRIVATE_KEY1>
+export TEST_PRIVATE_KEY2=<your TEST_PRIVATE_KEY2>
+```
+
+### Setup in Python
+
+Let's load services info and account info into Python `Config` and `Wallet` objects respectively.
+
+In your working console, run Python:
+```console
+python
+```
+
+In the Python console:
+```python
+# Create Ocean instance
+import os
+from ocean_lib.config import Config
+from ocean_lib.ocean.ocean import Ocean
+config = Config(os.getenv('OCEAN_CONFIG_FILE'))
+ocean = Ocean(config)
+
+# Create Alice's wallet
+import os
+from ocean_lib.web3_internal.wallet import Wallet
+alice_private_key = os.getenv('TEST_PRIVATE_KEY1')
+alice_wallet = Wallet(ocean.web3, alice_private_key, config.block_confirmations, config.transaction_timeout)
+```
+
+
+## 2. Publish Data NFT & Datatoken
+
+From here on, the code's is the same as local.
```

Comment:
can we add a sample here for CI purposes, if possible? I know it would be a duplicate and a DRY violation, but it would help with automated testing. Alternately let's have remote-based tests. We should explore that option as well: a remote-based test suite running periodically, like we run our lib viability/compatibility test.

What do you think, @trentmc?